config.default.yaml

```
@@ -270,10 +270,23 @@ industry:
   MWh_elec_per_tNH3_electrolysis: 1.17 # from https://doi.org/10.1016/j.joule.2018.04.017 Table 13 (air separation and HB)
   NH3_process_emissions: 24.5 # in MtCO2/a from SMR for H2 production for NH3 from UNFCCC for 2015 for EU28
   petrochemical_process_emissions: 25.5 # in MtCO2/a for petrochemical and other from UNFCCC for 2015 for EU28
-  HVC_primary_fraction: 1.0 #fraction of current non-ammonia basic chemicals produced via primary route
+  HVC_primary_fraction: 1. # fraction of today's HVC produced via primary route
+  HVC_mechanical_recycling_fraction: 0. # fraction of today's HVC produced via mechanical recycling
+  HVC_chemical_recycling_fraction: 0. # fraction of today's HVC produced via chemical recycling
+  HVC_production_today: 52. # MtHVC/a from DECHEMA (2017), Figure 16, page 107; includes ethylene, propylene and BTX
+  MWh_elec_per_tHVC_mechanical_recycling: 3. # estimate
```

```suggestion
  MWh_elec_per_tHVC_mechanical_recycling: 0.547 # from SI of https://doi.org/10.1016/j.resconrec.2020.105010, Table S5, for HDPE, PP, PS, PET. LDPE would be 0.756.
```
Adds the energy use of mechanical recycling. 

I have assumed energy demand for recycling of HPTE, PET, PS, PP as they are all very similar whereas LDPE is higher as well as more difficult to recycle.

    
    https://www.generalkinematics.com/blog/different-types-plastics-recycled/

---

Sounds good! Also for reference, another source [doi:10.1016/j.scitotenv.2017.05.278](http://dx.doi.org/10.1016/j.scitotenv.2017.05.278) with somewhat lower energy demand around 0.3 MWh/tPlastic. We can take the newer higher number.

---

Could we also check that the numbers from https://doi.org/10.1016/j.resconrec.2020.105010 are consistent with our chemical recycling assumptions? @brynpickering warned about reliability of Material Economics numbers in a private email.

---

The paper describes for routes of chemical recycling:

1) refinery feedstock: liquefy plastic -> use as a substitute for oil or naphtha in refinery and steam cracking processes
2) fuel production: gasification / pyrolysis -> gaseous / liquid fuels
3) monomer production: polyolefins -> monomers (e.g. ethylene) through thermal/catalytic pyrolysis
4) chemical upcycling

Route 3) comes closes to the Material Economics study (which uses pyrolysis and electric steam cracking and methanol synthesis to produce HVC from HVC for which they need 6.9 kWh/kg **electricity** with a yield of 0.91 kg/kg).

Route 3) in the paper following table S10 in the SI requires 3 MJ/kg (0.78 kWh/kg) of **heat** to turn HDPE into ethylene with a yield of 0.85 kg_ethylene / kg.

Not sure how conclusive this comparison is. Seems Material Economics numbers are a bit high?

*polyolefins: LDPE, HDPE, PP
*HVC: ethylene, propylene, polyolefins etc.